### PR TITLE
Send and Accept locale environment variables

### LINKED ISSUE #160: Possibly missing locale handling
I will freely admit my knowledge of locale in Linux is limited, hopefully others are more knowledgeable :-S

Whilst investigating some locale issues on our Ubuntu servers (14.04) I noticed that the ssh_config and sshd_config created by this cookbook lack the settings for sending and accepting locale, which differs from the default ubuntu/debian behaviour.

Specifically, sshd_config on debian would have:

`AcceptEnv LANG LC_*`

and ssh_config on debian would have:

`SendEnv LANG LC_*`

Can we add these back in, should we? (it won't fix our problem, but seems like an oversight)

### PR DESCRIPTION
**Pull request by mikemoate**
Fixes #160

### CODE REVIEW

**Comment by mikemoate** [comment]
@artem-sidorenko tests seem to fail on Oracle only, I'm not sure why, and I can't see how to view the specific failures in Travis?

**Comment by artem-sidorenko** [comment]
@mikemoate you just click the failed job and can see the log: https://travis-ci.org/dev-sec/chef-ssh-hardening/jobs/196915943

I looks totally unrelated to your changes and actually looks like master should be also broken. I will have a look

**Comment by artem-sidorenko** [comment]
it looks like this is related to the fix within train: https://github.com/chef/train/commit/6b5d58250c7fd5d9f8e21ca7e27ca400ffed6b5a

We have to add oracle platform to the ssh-baseline, I'll create a PR

**Comment by mikemoate** [comment]
OK thanks. I could see the Travis logs but not the details of the failing test cases.

Now I realise they are just very un-obvious (compared to the test passed lines which are highlighted red), maybe I'll give TravisCI some feedback :-)

**Comment by artem-sidorenko** [comment]
@mikemoate I guess this color problem is related somehow to the bash-color done by inspec. Usually it looks properly

**Comment by artem-sidorenko** [comment]
@mikemoate the https://github.com/dev-sec/ssh-baseline/pull/80 should fix the tests

File: templates/default/opensshd.conf.erb [hunk 1]
```diff
@@ -193,6 +193,9 @@ UseDNS <%= ((@node['ssh-hardening']['ssh']['server']['use_dns']) ? 'yes' : 'no'
 #ChrootDirectory none
 #ChrootDirectory /home/%u
 
+#Accept locale environment variables
+AcceptEnv LANG LC_* LANGUAGE
```
Comment (artem-sidorenko) [inline templates/default/opensshd.conf.erb]: Could we have it configurable too? Attribute like `['ssh-hardening']['ssh']['server']['accept_env']` with `['LANG', 'LC_*', 'LANGUAGE']` as default in the `attributes/default.rb` would be cool

File: spec/recipes/server_spec.rb [hunk 2]
```diff
@@ -64,6 +64,11 @@
     )
   end
 
+  it 'accepts locale environment variables' do
+    expect(chef_run).to render_file('/etc/ssh/sshd_config').
+      with_content('AcceptEnv LANG LC_* LANGUAGE')
```
Comment (artem-sidorenko) [inline spec/recipes/server_spec.rb]: If you implement it as attribute, could you please add the second test: to verify the attribute change? Good example is [here](https://github.com/dev-sec/chef-ssh-hardening/blob/74d0c6f97236464a31e303dcead8313f1b67438a/spec/recipes/server_spec.rb#L176-L193) for `allow_root_with_key` attribute

**Comment by artem-sidorenko** [comment]
Thank you for this PR! I've added some comments:)

**Comment by mikemoate** [comment]
@artem-sidorenko I think this covers everything you asked for now.

**Comment by artem-sidorenko** [comment]
@mikemoate looks good to me, could you please also cleanup the commit history by [squashing](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) the commits?

**Comment by mikemoate** [comment]
@artem-sidorenko squashed as requested.

**Comment by artem-sidorenko** [comment]
@mikemoate thank you!

**Pull request merged**